apps/condo/pages/phone/[number]/index.tsx

```diff
@@ -58,24 +62,89 @@ const ROW_MEDIUM_GUTTER: RowProps['gutter'] = [0, 40]
 const ROW_MEDIUM_SMALL_GUTTER: RowProps['gutter'] = [0, 24]
 const HINT_CARD_STYLE = { maxHeight: '3em' }
 const TICKET_SORT_BY = [SortTicketsBy.CreatedAtDesc]
-const PLUS_ICON_WRAPPER_CLASS = 'plusIconWrapper'
-const ADD_ADDRESS_TAB_KEY = 'addAddress'
 const HINTS_COL_PROPS: ColProps = { span: 24 }
 
-interface IClientContactProps {
-    phone: string
-    lastTicket: TicketType
-    contact?: ContactType
-    showOrganizationMessage?: boolean
-}
-
 type TabDataType = {
     type: ClientType
     property: Property
     unitName: string
     unitType: string
+    sectionType: string
+    sectionName: string
     organization: OrganizationType
-    contact?: Pick<ContactType, 'name' | 'id' | 'name'>
+    contact?: Pick<ContactType, 'name' | 'id'>
+}
+
+type ClientContactPropsType = {
+    phone: string
+    lastTicket: GetTicketsQuery['tickets'][number]
+    contact?: ContactType & { isEmployee?: boolean }
+    showOrganizationMessage?: boolean
+    type: string
+}
+
+const MAX_TABLE_SIZE = 20
+type TableTabs = 'address' | 'entrance' | 'home'
```

Comment:
Use `property` instead `home`.
And I would add more specificity to the values

```suggestion
type TabKey = 'contactPropertyTickets' | 'residentsEntranceTickets' | 'residentsPropertyTickets'
```
or something like that